flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/rules/FlinkStreamRuleSets.scala

```diff
@@ -382,7 +382,16 @@ object FlinkStreamRuleSets {
     PythonCalcSplitRule.PUSH_CONDITION,
     PythonCalcSplitRule.REWRITE_PROJECT,
     PythonMapRenameRule.INSTANCE,
-    PythonMapMergeRule.INSTANCE
+    PythonMapMergeRule.INSTANCE,
+    AsyncCalcSplitRule.SPLIT_CONDITION_REX_FIELD,
+    AsyncCalcSplitRule.SPLIT_PROJECTION_REX_FIELD,
+    AsyncCalcSplitRule.SPLIT_CONDITION,
+    AsyncCalcSplitRule.SPLIT_PROJECT,
+    AsyncCalcSplitRule.EXPAND_PROJECT,
+    AsyncCalcSplitRule.PUSH_CONDITION,
+    AsyncCalcSplitRule.REWRITE_PROJECT,
+    AsyncCalcSplitRule.NESTED_SPLIT,
```

Comment:
I don't know why Python people stopped the habit of adding inline comments. But a sentence per rule would help a lot to keep this class maintainable. Could you also add them to the Python ones? Esp. the "why" is interesting, e.g. "why one calc per split?"